### PR TITLE
LSWI-1552 recalibrate "empty" for std keg, bugs

### PR DESCRIPTION
**Pull request by 2bitoperations**

### CODE REVIEW

File: Tweeter.py [hunk 1]
```diff
@@ -19,7 +19,11 @@ def __init__(self, twAPIKey, twAPISecret, twAccessTok, twAccessTokS, scale, hour
 
     def run(self):
         self.logger.info("Starting twitter reporter, %d hours between reports." % self.hoursBetweenPosts)
-        time.sleep(60)
+        for i in range(60):
+            time.sleep(1)
+            if self.stopRequested:
+                return
```
Comment (matthewbogner) [inline Tweeter.py]: oh, good 'ole long running thread problem, eh?

Comment (2bitoperations) [inline Tweeter.py]: yea.... I was pretty sloppy w/ this code....

File: config.example.yml [hunk 2]
```diff
@@ -8,8 +8,8 @@ twitter:
   accessTokenSecret: ACCESSTOKENSECRET
   hoursBetweenPosts: 1
 keg:
-  emptyKg: 13.5
-  fullKg: 72.8
+  emptyKg: 19.4
+  fullKg: 78.7
```
Comment (matthewbogner) [inline config.example.yml]: cheating

**Comment by matthewbogner** [comment]
ship it

**Pull request merged**